app/routes/umd_tree_cover_loss/raster_tiles.py

```diff
@@ -56,7 +56,38 @@ async def umd_tree_cover_loss_raster_tile(
     background_tasks: BackgroundTasks,
 ) -> Response:
     """
-    UMD tree cover loss raster tile.
+    UMD Tree Cover Loss raster tile.
+
+    There are two types of TCL tiles available here: True color tiles and encoded tiles.
+    The true color tiles have been pre-rendered to appear the same way TCL appears on
+    globalforestwatch.org (i.e. the tiles show pink pixels to indicate tree cover loss).
+    The caveat with the true color tiles is that ANY tree cover loss over the history of
+    the data is shown as a pink pixel. There's no way to filter out pixels which fall in
+    or out of a range of years like there is in the map interface, because that
+    information is no longer in the data!
+    The encoded tiles, on the other hand, still contain the year information but are not
+    ready to be displayed as-is. They can/need to be interpreted and rendered according to
+    the needs of the consumer of the data.
+    Note that both types of tiles are thresholded by Tree Cover Density, which means that
+    one needs to know the TCD percentage which one wants to consider necessary to
+    indicate tree cover loss of interest. A small example: If one wants data which
+    indicates loss in areas that have a tree cover density of 75% or more, one would
+    specify a TCD of 75. The allowable thresholds can be seen below in the query
+    parameters.
+    So if you want
+
+    True color tiles (i.e. pink pixels as they appear on the map at globalforestwatch.org) thresholded by tree cover density
+        Specify the style query parameter
+        For example: https://tiles.globalforestwatch.org/umd_tree_cover_loss/v1.10/dynamic/1/1/0.png?style=tcd_10
+
+    True color tiles filtered by start and/or end year, thresholded by tree cover density
+        Specify the tcd query parameter and optionally the start/end year
+        For example: https://tiles.globalforestwatch.org/umd_tree_cover_loss/v1.10/dynamic/1/1/0.png?tcd=10&start_year=2016
+
+    Encoded tiles, so you can filter and style on your own
```

Comment:
Just to follow the pattern of the previous two, maybe change to:

Encoded tiles (so you can filter and style on your own), thresholded by tree cover density